src/Install/Steps/WriteSettings.php

```diff
@@ -62,6 +62,7 @@ private function getDefaults()
             'custom_less' => '',
             'default_locale' => 'en',
             'default_route' => '/all',
+            'enable_user_pass_auth' => '1',
```

Comment:
I would prefer the word "password" to be written in full in the setting name. "user" could be dropped.

Something like "enable_password_auth" or even "enable_password_authentication".

I feel like shortening the name can only lead to confusion and doesn't provide any real performance gain.

Reply:
I've changed it to enable_password_auth throughout, I agree that's a lot clearer. Thanks!